crates/biome_rule_options/src/use_consistent_arrow_return.rs

```diff
@@ -1,6 +1,45 @@
+use biome_deserialize::{
+    DeserializableValidator, DeserializationContext, DeserializationDiagnostic,
+};
 use biome_deserialize_macros::Deserializable;
 use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Deserializable, Eq, PartialEq, Serialize, Default)]
+#[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
+#[serde(rename_all = "camelCase")]
+pub enum UseConsistentArrowReturnStyle {
+    #[default]
+    AsNeeded,
+    Always,
+    Never,
+}
+
 #[derive(Default, Clone, Debug, Deserialize, Deserializable, Eq, PartialEq, Serialize)]
 #[cfg_attr(feature = "schema", derive(schemars::JsonSchema))]
 #[serde(rename_all = "camelCase", deny_unknown_fields, default)]
-pub struct UseConsistentArrowReturnOptions {}
+#[deserializable(with_validator)]
+pub struct UseConsistentArrowReturnOptions {
+    pub style: UseConsistentArrowReturnStyle,
+    pub require_for_object_literal: bool,
```

Comment:
Options must be documented here, because this becomes part of the schema and is used on the website too.